docker-compose.yml

```diff
@@ -57,6 +61,7 @@ services:
       dockerfile: Dockerfile
     container_name: handshake
     restart: on-failure
+    command: hnd --log-console=false
```

Comment:
Shouldn't this be `command: hsd --log-console=false`

Reply:
nope, handshake docker container uses `hsd` (handshake daemon) `ENTRYPOINT ["hsd"]`